src/tasks/captureScreenshots.ts

```diff
@@ -1,6 +1,6 @@
 import { ListrTask, ListrRendererFactory } from 'listr2';
 import { Context } from '../types.js'
-import { captureScreenshots } from '../lib/screenshot.js'
+import { captureScreenshots, captureScreenshotsConcurrent } from '../lib/screenshot.js'
```

Comment:
If possible can we remove the old unused functions ? Since in each case the newly created function would be utilized.

Reply:
let it be there till next release, once codes get stable will remove that @parthlambdatest

Reply:
Sure